test/Integration.spec.ts

```diff
@@ -55,6 +55,22 @@ describe('Integration test', function() {
     await runTestProject('mjs-ts');
   });
 
+  describe('esbuild', () => {
```

Comment:
It might be better to find a way of running all tests on a minified version of the library long-term.

Reply:
Agreed! Filed a ticket for this.